content/blog/jt15s/2020-12-31_hello_from_the_haiku_promotion_team.md

```diff
@@ -0,0 +1,74 @@
+Strategies such as making sure user queries are answered as well as spreading the word about new apps and releases is a good first start, but we are considering various other methods too.
+
+### App Developers
+If you follow technology news, you might have seen a mention of Medo, the recently-released 4K editor clocking in at only 1.44MB! This number alone highlights one of Haiku's many development advantages: our apps are small and light - even if you have a project with lots of features crammed into it, on Haiku the file size will likely be very reasonable. Add that to the fact that we have HaikuPorts, an organised porting system, that we use C++, one of the most popular progamming languages in the world, and that we are compatible with frameworks like Qt, a popular UI frontend framework, and you get the development environment that is Haiku. We want to attract more app developers, whether that's developers who want to port their existing app over to Haiku, or new developers who wish to make an app in Haiku that they feel is needed.
+Additionally, we want to make any existing ports of FOSS projects to Haiku official platforms - this means having Haiku listed as an officially supported platform on the project's website. By getting Haiku listed on projects such as KDE, more people will be aware of Haiku and the fact that Haiku has a multitude of apps available to download.
+
+### Researchers
+Surprisingly, Haiku is also an excellent system for research purposes. Let me explain further. Researchers don't need the stability and reliability that businesses do, which means Haiku is a possible research system for them. Furthermore, it is possible to make invasive and disruptive changes to Haiku without having to dodge security restrictions imposed by the system, or having to "hack" their way into the system. In the past, Haiku has had successful collaborations with the Auckland University, where a group working on UI/UX research experimented with Haiku. 
+They produced Stack&Tile as well as the [Auckland Layout Engine (ALE)](https://www.researchgate.net/publication/262402078_The_Auckland_Layout_Editor_An_improved_GUI_layout_specification_process) for laying out user interfaces using a constraint solver. Apart from ALE, the group also had other projects which were not implemented into Haiku, such as a tool to auto-generate user documentation from user interface code as well as a way to dynamically modify the layout of a running window to optimise it for a specific workflow.
+
+Apart from Auckland University, mmu_man, a member of the Haiku Community, also published a research paper about the interoperability of filesystem attributes.
```

Comment:
The paper about filesystem attributes: https://dcpapers.dublincore.org/pubs/article/view/3633/1859

Also an article about using QEMU for operating system development at the QEMU user forum: http://adt.cs.upb.de/quf/quf11/QUF11-papers/quf2011-11.pdf